config/prow/cluster/build_kubernetes-external-secrets_customresource.yaml

```diff
@@ -13,3 +13,16 @@ spec:
   - key: azure-cred
     name: credentials
     version: latest
+---
+apiVersion: kubernetes-client.io/v1
+kind: ExternalSecret
+metadata:
+  name: snyk_token
+  namespace: test-pods
+spec:
+  backendType: gcpSecretsManager
+  projectId: kubernetes-upstream # gsm project id
```

Comment:
verify if this projectId is correct

/hold

Reply:
cc: @spiffxp @cjwagner is this correct project we'd want to have secrets into?

Reply:
```suggestion
  projectId: k8s-infra-prow-build-trusted # gsm project id
```

Reply:
done
/unhold

Reply:
@spiffxp : can this CR reside in this file ? or should I move to https://github.com/kubernetes/k8s.io/tree/main/infra/gcp/clusters/projects/k8s-infra-prow-build-trusted/prow-build-trusted/resources ?